doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -206,10 +206,13 @@ In order for the storage solution(s) to be conformant with the Reference Archite
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.stg.001`||||
-|`ra2.stg.002`||||
-|`ra2.stg.003`||||
-|`ra2.stg.004`||||
+|`ra2.stg.001`| Ephemeral Storage | An implementation must support ephemeral storage, for the unpacked container images to be stored and executed from, as a directory in the filesystem on the worker node on which the container is running <br>See the [Container runtimes](#4.4) section above for more information on how this meets the requirement for ephemeral storage for containers <br>Ephemeral storage exists for the life of a container<br>Kubernetes Volumes attached using a Volume plugin can also be used to deliver Ephemeral Storage (may require additional tasks to ensure deletion of ephemeral storage on the shutting down of a container) ||
```

Comment:
I'd also argue the following is unnecessary here and should be part of the chapter03.md content:

> Ephemeral storage exists for the life of a container<br>Kubernetes Volumes attached using a Volume plugin can also be used to deliver Ephemeral Storage (may require additional tasks to ensure deletion of ephemeral storage on the shutting down of a container).

Reply:
Agree with Tom

Reply:
@tomkivlin @xmulligan 

Deleted content that belongs to RA Chapter 3:

- from ra2.stg.001: "Ephemeral storage exists for the life of a container.<br>Kubernetes Volumes attached using a Volume plugin can also be used to deliver Ephemeral Storage (may require additional tasks to ensure deletion of ephemeral storage on the shutting down of a container)" 

- from from ra2.stg.002: "This additional storage, that is attached to a container using a Kubernetes Volume, does not live and die with the container but instead follows the lifecycle of the Pod that the container is a part of. " 

- from ra2.stg.005: "The implementation may utilise either an In-tree or Out-of-tree type of Volume plugin.<br>In-tree plugins are built, linked, compiled and shipped with the core Kubernetes binaries and, thus, require the core Kubernetes code to be changed whenever a new backend storage system needs to be supported.<br>Out-of-tree plugins allow new backend storage systems to be supported without any changes to the core Kubernetes code." 

- from ra2.stg.007: "Storage Classes allow a consumer of the Kubernetes platform to request Persistent Storage using a Persistent Volume Claim and for a Persistent Volume to be dynamically created based on the "class" that has been requested. This avoids having to grant `create`/`update`/`delete` permissions in RBAC to PersistentVolume resources, which are cluster-scoped rather than namespace-scoped (meaning an identity can manage all PVs or none)."